### PR TITLE
Add auoms package

### PR DESCRIPTION
**Pull request by anphel31**
###### Merge Checklist  

**All** boxes should be checked before merging the PR *(just tick any boxes which don't apply to this PR)*
- [x] The toolchain has been rebuilt successfully (or no changes were made to it)
- [x] The toolchain/worker package manifests are up-to-date
- [x] Any updated packages successfully build (or no packages were changed)
- [x] All package sources are available
- [x] cgmanifest files are up-to-date and sorted (`./cgmanifest.json`, `./toolkit/tools/cgmanifest.json`, `./toolkit/scripts/toolchain/cgmanifest.json`)
- [x] All source files have up-to-date hashes in the `*.signatures.json` files
- [x] `sudo make go-tidy-all` and `sudo make go-test-coverage` pass
- [x] Documentation has been updated to match any changes to the build system
- [x] Ready to merge

---

###### Summary 

This change adds the auoms package to CBL-Mariner core

###### Change Log  

- Add auoms spec
- Update cgmanifest.json with auoms sources
- Update LICENSES-MAP file with auoms (as well as omi, which was recently added but missing from this file)

###### Does this affect the toolchain?  

YES
**NO**

###### Test Methodology

- Local package build
sudo make build-packages PACKAGE_BUILD_LIST="auoms" PACKAGE_REBUILD_LIST="auoms" -j$(nproc)

### CODE REVIEW

File: SPECS/auoms/auoms.spec [hunk 1]
```diff
@@ -0,0 +1,193 @@
...
+/opt/microsoft/auoms/bin/auoms
+/opt/microsoft/auoms/bin/auomsctl
+/var/opt/microsoft/auoms
+/var/opt/microsoft/auoms/data
+/var/opt/microsoft/auoms/data/outputs
+
+%changelog
+* Thu Oct 22 2020 Andrew Phelps <anphel@microsoft.com> 2.2.5-1
+- Initial CBL-Mariner version.  
+* Tue Sep 15 2020 Henry Li <lihl@microsoft.com> 2.3.0-1
```
Comment (henryli001) [inline SPECS/auoms/auoms.spec:192]: Can you please change the version number to 2.2.5-1? 2.3.0-1 was taken out from the diagnostics repo and this information was mistakenly not updated.

**Review by henryli001 — APPROVED** [review]

File: SPECS/auoms/auoms.spec (hunk 1 above)
Comment (anphel31) [reply SPECS/auoms/auoms.spec]: Fixed

File: SPECS/auoms/auoms.spec [hunk 2]
```diff
@@ -0,0 +1,193 @@
+%define      debug_package %{nil}
+
+Summary:        Auditd plugin that forwards audit events to OMS Agent for Linux
+Name:           auoms
+Version:        2.2.5
+Release:        1%{?dist}
+License:        MIT
```
Comment (jslobodzian) [inline SPECS/auoms/auoms.spec:7]: Are all of these sources MIT?

File: SPECS/auoms/auoms.spec [hunk 3]
```diff
@@ -0,0 +1,193 @@
...
+/opt/microsoft/auoms/bin/auomsctl
+/var/opt/microsoft/auoms
+/var/opt/microsoft/auoms/data
+/var/opt/microsoft/auoms/data/outputs
+
+%changelog
+* Thu Oct 22 2020 Andrew Phelps <anphel@microsoft.com> 2.2.5-1
+- Initial CBL-Mariner version.  
+* Tue Sep 15 2020 Henry Li <lihl@microsoft.com> 2.2.5-0
+- Initial build. First version
```
Comment (jslobodzian) [inline SPECS/auoms/auoms.spec:193]: > bui [](start = 9, length = 4)

Change this to read: Initial CBL-Mariner Version.

File: SPECS/auoms/auoms.spec [hunk 4]
```diff
@@ -0,0 +1,193 @@
...
+%{_sysconfdir}/init.d/auoms
+%{_sysconfdir}/opt/microsoft/auoms
+%{_sysconfdir}/opt/microsoft/auoms/auoms.conf
+%{_sysconfdir}/opt/microsoft/auoms/auomscollect.conf
+%{_sysconfdir}/opt/microsoft/auoms/example_output.conf
+%{_sysconfdir}/opt/microsoft/auoms/outconf.d
+%{_sysconfdir}/opt/microsoft/auoms/rules.d
+/opt/microsoft/auoms
+/opt/microsoft/auoms/LICENSE
+/opt/microsoft/auoms/THIRD_PARTY_IP_NOTICE
```
Comment (jslobodzian) [inline SPECS/auoms/auoms.spec:179]: there isn't a %license section in the spec file.   the LICENSE and THIRD_PARTY_IP_NOTICE are probably the correct content...but...i don't know if its going to the right place.

File: SPECS/auoms/auoms.spec (hunk 2 above)
Comment (anphel31) [reply SPECS/auoms/auoms.spec:7]: auoms is MIT, Pal is MIT, msgpack-c is Boost license, RapidJSON is MIT and JSON license.

Comment (anphel31) [reply SPECS/auoms/auoms.spec:7]: The THIRD_PARTY_IP_NOTICE file listed below includes the MIT license and the Boost license for msgpack-c. The limited use of rapidjson in this package only requires the MIT license.

---
In reply to: [510529327](https://github.com/microsoft/CBL-Mariner/pull/258#discussion_r510529327) [](ancestors = 510529327)

File: SPECS/auoms/auoms.spec (hunk 4 above)
Comment (anphel31) [reply SPECS/auoms/auoms.spec]: fixed

**Review by jslobodzian — APPROVED** [review]
:shipit:

**Pull request merged**